po/zh.po

```diff
@@ -1324,29 +1303,24 @@ msgid "Go to a random unread article"
 msgstr "转到随机一篇未读文章"
 
 #: src/keymap.cpp:126
-#, fuzzy
 msgid "Open URL of article, or entry in URL view. Mark read"
-msgstr "在浏览器里打开文章"
+msgstr "在链接视图(URL view)中打开文章或某一条目(entry)。标记为已读"
```

Comment:
They are both translated and I placed the original English texts after their translations with `()`.

`链接视图` for `URL view` and `条目` for `entry` are not frequently used in Chinese, so it may be confusing when first encountered. From my point of view, I would like to check over the docs to find the original usage of these (we can say terminologies?). Since Newsboat hasn't had docs in other languages, I think it's better to help the user find the terms in the English docs instead of getting stuck in the Chinese-English matching.

The same question goes to `query feeds` which is more like a term. There is a [chapter](https://newsboat.org/releases/2.33/docs/newsboat.html#_query_feeds) in the docs to explain it. If I translate it like this: `查询` for `query` and `种子` for `feeds`,  so `查询种子`, it may be more confusing than `链接视图` and `条目` above. I haven't decided how to translate it yet.